rubin_sim/data/rs_download_data.py

```diff
@@ -154,8 +170,12 @@ def rs_download_data():
             file_size = int(r.headers.get("Content-Length", 0))
             if file_size < 245:
                 warnings.warn(f"{url} file size unexpectedly small.")
-            block_size = 1024 * 1024  # download this size chunk at a time; reasonable guess
-            progress_bar = tqdm(total=file_size, unit="iB", unit_scale=True, disable=args.tdqm_disable)
+            block_size = (
+                1024 * 1024
+            )  # download this size chunk at a time; reasonable guess
```

Comment:
In general, it's better to have comments on lines by themselves, unless they're very short. In this case in particular, it looks *much* better if you do so.

```suggestion
            # download this size chunk at a time; reasonable guess
            block_size = 1024 * 1024
```